src/main/java/org/broadinstitute/hellbender/utils/genotyper/AlleleLikelihoods.java

```diff
@@ -144,14 +148,19 @@ public AlleleLikelihoods(final SampleList samples,
 
         referenceAlleleIndex = findReferenceAllele(alleles);
         sampleMatrices = (LikelihoodMatrix<EVIDENCE,A>[]) new LikelihoodMatrix[sampleCount];
+        numberOfEvidences = new int[sampleCount];
```

Comment:
`numberOfEvidences = new IndexRange(0, sampleCount).mapToInteger(s -> evidenceBySampleIndex.get(s).size())`, if you are so inclined.

Reply:
What about this?
```
numberOfEvidences = IntStream.range(0, sampleCount)
          .map(i -> evidenceBySampleIndex.get(i).size())
          .toArray();
```

Reply:
That works, too.